### PR TITLE
Adding Guest wifi management and extra device properties

### PR DESCRIPTION
**Pull request by mikeluyten**
#112 
Appologies for the new PR, had to clear out some bad commit history

### CODE REVIEW

File: tplinkrouterc6u/client/c6u.py [hunk 1]
```diff
@@ -264,11 +268,78 @@ def set_wifi(self, wifi: Connection, enable: bool) -> None:
...
+        guest_wifi.guest_2g_portal_password = data.get('guest_2g_portal_password')
+        guest_wifi.guest_5g_enable = self._str2bool(data.get('guest_5g_enable'))
+        guest_wifi.guest_5g_ssid = data.get('guest_5g_ssid')
+        guest_wifi.guest_5g_encryption = data.get('guest_5g_encryption')
+        guest_wifi.guest_5g_psk_key = data.get('guest_5g_psk_key')
+        guest_wifi.guest_5g_portal_enable = self._str2bool(data.get('guest_5g_portal_enable'))
+        guest_wifi.guest_5g_portal_password = data.get('guest_5g_portal_password')
+        return guest_wifi
+
+    def set_guest_wifi_password(self, password: str, band: str = '2g5g') -> None:
```
Comment (AlexandrErohin) [inline tplinkrouterc6u/client/c6u.py]: set_wifi already do it - no need to add set_guest_wifi_password

File: test/test_client_c6u.py [hunk 2]
```diff
@@ -10,10 +10,50 @@
...
+        self.assertEqual(guest_wifi_info.guest_2g_ssid, "guest_2g")
+        self.assertEqual(guest_wifi_info.guest_2g_encryption, "psk")
+        self.assertEqual(guest_wifi_info.guest_2g_psk_key, "password")
+        self.assertEqual(guest_wifi_info.guest_2g_portal_enable, False)
+        self.assertEqual(guest_wifi_info.guest_5g_enable, False)
+        self.assertEqual(guest_wifi_info.guest_5g_ssid, "guest_5g")
+        self.assertEqual(guest_wifi_info.guest_5g_encryption, "none")
+        self.assertEqual(guest_wifi_info.guest_5g_psk_key, "")
+
     def test_get_status(self) -> None:
```
Comment (AlexandrErohin) [inline test/test_client_c6u.py]: it is better to add to old test only checks that your new fields are None for case when router doesnt have these fields. And add a new test that checks all fields values with your new fields.
